index.html

```diff
@@ -0,0 +1,143 @@
+          <option value="job">Full Time Job</option>
+          <option value="learner">Full Time Learner</option>
+          <option value="preferNo">Prefer not to say</option>
+          <option value="other">Other</option>
+        </select>
+      </div>
+
+      <div class="form-group">
+        <p>Would you recommend freeCodeCamp to a friend?</p>
+        <label class="form-label">
+          <input class="form-check-input" name="user-recommend" value="definitely" type="radio" class="input-radio" checked="">Definitely</label>
+        <label>
+          <input name="user-recommend" value="maybe" type="radio" class="form-check-input">Maybe</label>
+
+        <label><input name="user-recommend" value="not-sure" type="radio" class="form-check-input">Not sure</label>
+      </div>
+
+      <div class="form-group">
+        <p>
+          What is your favorite feature of freeCodeCamp?
+        </p>
+        <select id="most-like" name="mostLike" class="form-control" required="">
+          <option disabled="" selected="" value="">Select an option</option>
+          <option value="challenges">Challenges</option>
+          <option value="projects">Projects</option>
+          <option value="community">Community</option>
+          <option value="openSource">Open Source</option>
+        </select>
+      </div>
+
+      <div class="form-group form-check">
+        <p>
+          What would you like to see improved?
+          <span class="clue">(Check all that apply)</span>
+        </p>
+
+        <label><input name="prefer" value="front-end-projects" type="checkbox" class="form-check-input">Front-end
+          Projects</label><br>
+        <label>
+          <input name="prefer" value="back-end-projects" type="checkbox" class="form-check-input">Back-end
+          Projects</label><br>
+        <label><input name="prefer" value="data-visualization" type="checkbox" class="form-check-input">Data
+          Visualization</label><br>
+        <label><input name="prefer" value="challenges" type="checkbox" class="form-check-input">Challenges</label><br>
+        <label><input name="prefer" value="open-source-community" type="checkbox" class="form-check-input">Open Source
+          Community</label><br>
+        <label><input name="prefer" value="gitter-help-rooms" type="checkbox" class="form-check-input">Gitter help
+          rooms</label><br>
+        <label><input name="prefer" value="videos" type="checkbox" class="form-check-input">Videos</label><br>
+        <label><input name="prefer" value="city-meetups" type="checkbox" class="form-check-input">City Meetups</label><br>
+        <label><input name="prefer" value="wiki" type="checkbox" class="form-check-input">Wiki</label><br>
+        <label><input name="prefer" value="forum" type="checkbox" class="form-check-input">Forum</label><br>
+        <label><input name="prefer" value="additional-courses" type="checkbox" class="form-check-input">Additional
+          Courses</label><br>
```

Comment:
Please make sure that the inputs with each label are in a ul tag, each of them in a li tag.